doc_src/CreatingBackends.md

```diff
@@ -3,6 +3,10 @@
 
 See `Hammer.Backend.ETS` for a realistic example of a Hammer Backend module.
 
+
+Backends for Hammer are expected to implement the [Hammer.Backend](/hammer/Hammer.Backend.html) behaviour.
```

Comment:
This link will be broken in many cases. It is better in my experience to have full urls.